server/api/api.yaml

```
@@ -109,6 +109,13 @@ paths:
           example: server down
           schema:
             type: string
+        - name: channel_id
+          in: query
+          description: The returned list will contain only the playbook run associated with this channel ID.
```

nit: feel free to ignore
```suggestion
          description: The returned list will contain only the playbook runs associated with this channel ID.
```